tests/unit/test_clidriver.py

```diff
@@ -766,15 +768,15 @@ def test_idempotency_token_is_not_required_in_help_text(self):
     @mock.patch('awscli.clidriver.distro.major_version', return_value='1')
     def test_user_agent_for_linux(self, *args):
         driver = create_clidriver()
-        expected_user_agent = 'source/x86_64.amzn.1'
+        expected_user_agent = 'md/installer#source md/distrib#amzn.1'
         self.assertEqual(expected_user_agent,
                          driver.session.user_agent_extra)
 
     def test_user_agent(self, *args):
         machine = platform.machine()
         driver = create_clidriver()
         user_agent_extra_pattern = re.compile(
-            fr'^source/{machine}(\.[a-z_]+)?(\.[0-9]+)?$'
+            r'^md/installer#source'
```

Comment:
This was failing specifically on Ubuntu since we split the installer source and the arch/distribution into separate components.

Reply:
`{machine}` is no longer needed because its no longer part of the user agent extra.